src/qtt/simulation/dotsystem.py

```diff
@@ -270,9 +270,6 @@ def __init__(self, name='dotsystem', ndots=3, **kwargs):
         self.ndots = ndots
         self.temperature = 0
 
-        self.makevars = qtt.utilities.tools.deprecated(self.make_variables)
-        self.makevarsM = qtt.utilities.tools.deprecated(self._make_variable_matrices)
-
```

Comment:
There are 2 more instances of @qtt.utilities.tools.deprecated in this module (getHn, defaultVmatrix). why are they not removed?

A search also shows instances of @qtt.utilities.tools.deprecated or @qtt.utilities.tools.rdeprecated in:
generic.py show2Dimage
data.py 2 instances
scans.py 2 instances
functions.py _cost_double_gaussian
instrument_storage.py, 2 instances
virtual_gates.py 1 instance

Reply:
The `rdeprecated` throws an exception, so we can be sure everyone using it should be aware of the deprecation. The `deprecated` only prints a warning, so I wanted to be on the safe side.
I will remove most `deprecated` code as well, or convert from `deprecated` to `rdeprecated`